sklearn/metrics/tests/test_regression.py

```diff
@@ -566,7 +566,7 @@ def test_mean_pinball_loss_on_constant_predictions(distribution, target_quantile
         # Check that the loss of this constant predictor is greater or equal
```

Comment:
In the file `sklearn/metrics/tests/test_common.py`, I think that you need to add the new metric into the dictionary `array_api_metric_checkers` such that the common test for array api are run on them.

Reply:
Yes, I forgot about the `array_api_metric_checkers` dictionary. I've fixed it now. Thanks for catching that!